README.md

```diff
@@ -1,18 +1,62 @@
-# why?
+# browser-previews flake for NixOS users
 
-google-chrome-beta and google-chrome-dev were not getting maintained in nixpkgs,
-and were dropped in https://github.com/NixOS/nixpkgs/pull/261870.
+This flake provides the latest Chrome stable, beta and dev releases. The flake is updated regularly to ensure fast and up to date releases, which would otherwise not be possible in [nixpkgs](https://github.com/NixOS/nixpkgs).
 
+## Why?
 
-# how?
+`google-chrome-beta` and `google-chrome-dev` were not getting maintained in nixpkgs,
+and were dropped in [NixOS/nixpkgs#261870](https://github.com/NixOS/nixpkgs/pull/261870).
```

Comment:
thought: the extra markdown isn't necessary, it already displays with the short name when rendered:

![image](https://github.com/r-k-b/browser-previews/assets/2717397/5c076fab-a67e-4bf2-b7d5-e19965a0c9d1)

Reply:
Oh... I was getting a warning on my markdown linter that this is a naked link, so I added the label.